components/section/TheMenu.vue

```diff
@@ -122,7 +122,12 @@
 </template>
 
 <script>
+import SocialLinks from "/components/SocialLinks.vue";
+
 export default {
+    components: {
```

Comment:
I fixed this before 😅 https://github.com/devedmonton/DES-Website/commit/da8db312da727d1155dece777ea00357694d6f1f wonder why it got changed back.

Reply:
ohhhh probably a merge conflict mess up with that bigger pr